.eslintrc.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable import/no-commonjs */
+const defaultRules = {
+  'no-continue': 'off',
+  'valid-jsdoc': 'off',
+  'require-await': 'off',
+  'import/extensions': [
+    'error',
+    {
+      ignorePackages: true,
+      pattern: {
+        js: 'always',
+        ts: 'never',
+      },
+    },
+  ],
+};
+module.exports = {
+  overrides: [
+    {
+      files: ['**/*.js'],
+      extends: ['algolia', 'algolia/jest'],
+      rules: defaultRules,
+    },
+    {
+      files: ['**/*.ts'],
+      extends: ['algolia', 'algolia/jest', 'algolia/typescript'],
+      parser: '@typescript-eslint/parser',
+      parserOptions: {
+        sourceType: 'module',
+        project: './tsconfig.json',
+      },
+      rules: {
+        ...defaultRules,
+        'consistent-return': 'off',
+        'no-dupe-class-members': 'off',
+      },
+    },
+  ],
```

Comment:
```suggestion
  rules: defaultRules,
  extends: ['algolia', 'algolia/jest'],
  overrides: [
    {
      files: ['**/*.ts'],
      extends: ['algolia', 'algolia/jest', 'algolia/typescript'],
      parser: '@typescript-eslint/parser',
      parserOptions: {
        sourceType: 'module',
        project: './tsconfig.json',
      },
      rules: {
        'consistent-return': 'off',
        'no-dupe-class-members': 'off',
      },
    },
  ],
```

this works too, no? And which files are .js? do we need that config still?